### PR TITLE
[LeftPane] adding new props to screen and adding new menu button

### PR DESCRIPTION
**Pull request by JoshuaLai**

### CODE REVIEW

File: Chat/Chat.csproj [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <DefaultItemExcludes>$(DefaultItemExcludes);$(SpaRoot)node_modules\**</DefaultItemExcludes>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'Web' " />
```
Comment (armansabaa) [inline Chat/Chat.csproj]: Do we need to checkin those changes?

File: Chat/ClientApp/src/components/LeftPane.tsx [hunk 2]
```diff
@@ -3,7 +3,13 @@ import { DefaultButton, Icon, IconButton, Pivot, PivotItem, Stack, StackItem } f
...
 
-export default (props: any): JSX.Element => {
+export interface LeftPaneProps {
+  onTabClicked(): void;
+}
+
+export default (props: LeftPaneProps): JSX.Element => {
+  const { onTabClicked } = props;
+
   return (
```
Comment (armansabaa) [inline Chat/ClientApp/src/components/LeftPane.tsx:13]: Have you pulled in latest main? There was 2 buttons here I believe :)

File: Chat/ClientApp/src/components/MainArea.tsx [hunk 3]
```diff
@@ -3,10 +3,17 @@ import { Image } from '@fluentui/react';
...
-export default (props: any): JSX.Element => {
+export interface MainScreenProps {
+  updateContent(content: string): void;
+}
+
+export default (props: MainScreenProps): JSX.Element => {
   const imageProps = { src: defaultImg.toString() };
-  return (
-    <div className={staticAreaStyle}>
+  const { updateContent } = props;
```
Comment (armansabaa) [inline Chat/ClientApp/src/components/MainArea.tsx]: Not used?

File: Chat/ClientApp/src/components/MainArea.tsx [hunk 4]
```diff
@@ -3,10 +3,17 @@ import { Image } from '@fluentui/react';
...
+
+  const getComponent = () => {
```
Comment (armansabaa) [inline Chat/ClientApp/src/components/MainArea.tsx:15]: From JS perspective, this change is equivalent to the previous code state.

File: Chat/Chat.csproj (hunk 1 above)
Comment (JoshuaLai) [reply Chat/Chat.csproj]: Ah this a good point. I think these changes were made when I just opened in visual studio

File: Chat/ClientApp/src/components/LeftPane.tsx (hunk 2 above)
Comment (JoshuaLai) [reply Chat/ClientApp/src/components/LeftPane.tsx:13]: Yes there are still buttons there for `Home` and `CalendarAgenda`

File: Chat/ClientApp/src/components/MainArea.tsx (hunk 3 above)
Comment (JoshuaLai) [reply Chat/ClientApp/src/components/MainArea.tsx]: No not yet. I will have it in a part 2 PR but going to use it to change what `MainArea.tsx` displays I think. Or at least that is the plan I had. Open to thoughts

File: Chat/ClientApp/src/components/MainArea.tsx (hunk 4 above)
Comment (JoshuaLai) [reply Chat/ClientApp/src/components/MainArea.tsx:15]: Yep I agree. As per comment above I will introduce the ability to change what displays on here

**Comment by JoshuaLai** [comment]
Added some logic to change the main page when we click on a tab item 
Screen Shot 2021-06-08 at 4 42 08 PM
Screen Shot 2021-06-08 at 4 42 12 PM

**Review by minnieliu — APPROVED** [review]

**Pull request merged**